src/logic/safe/transactions/multisend.ts

```diff
@@ -34,3 +37,13 @@ export const getMultiSendJoinedTxs = (txs: Transaction[]): string => {
 
   return `0x${joinedTxs}`
 }
+
+export const isSupportedMultiSendAddress = (txInfo: TransactionInfo): boolean => {
+  const toAddress = isCustomTxInfo(txInfo) ? txInfo.to.value : ''
+  const multiSendAddress = getMultisendContractAddress()
+
+  return sameString(multiSendAddress, toAddress)
+}
+
+export const isSupportedMultiSendCall = (txInfo: TransactionInfo): txInfo is MultiSend =>
+  isSupportedMultiSendAddress(txInfo) && isCustomTxInfo(txInfo) && txInfo.methodName === 'multiSend'
```

Comment:
```suggestion
   isCustomTxInfo(txInfo) && isSupportedMultiSendAddress(txInfo) && txInfo.methodName === 'multiSend'
```

Otherwise there are Typescript errors in `TxSummary.ts`